packages/shared/schemas/src/lib/game/pack_validation.ts

```
@@ -0,0 +1,348 @@
+
+  return { packId, errors, warnings, autoFixes };
+};
+
+/**
+ * Checks whether a string contains a URL scheme or path traversal that
+ * would make it a network origin or filesystem escape.
+ * Used by the hostile-manifest test (AC-2).
+ */
+export const isHostileString = (s: string): boolean => {
+  return isAbsoluteUrl(s) || hasPathTraversal(s) || isDataScheme(s) || isJavaScriptScheme(s);
+};
```

Biome (2.5.7)

[error] The body of this arrow function contains a single return statement.

(lint/style/useConsistentArrowReturn)